### DISCUSSION TITLE
Browser i18n? (internationalization)

### DISCUSSION THREAD

**Discussion by johanbove** [beakerbrowser/beaker#1657] [General] [answered]
Currently there is no way a user can set their preferred browser language or use Beaker in other languages than English. 

What are the plans for make Beaker multi - lingual?

Adding a language preference for websites would allow to create more user-friendly web apps. See https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/language

It would also need to support right to left reading and support for non-Latin characters.

**Comment by rzelnik** [comment, marked as answer]
I am also interested in i18n of Beaker - see https://github.com/beakerbrowser/beaker/issues/1543

**Reply by pfrazee** [reply]
Yeah, track that issue for updates.
